scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/client/BigQueryConfig.scala

```diff
@@ -70,6 +72,14 @@ object BigQueryConfig {
   def readTimeoutMs: Option[Int] =
     BigQuerySysProps.ReadTimeoutMs.valueOption.map(_.toInt)
 
+  def impersonationInfo: Option[ImpersonationInfo] = {
+    BigQuerySysProps.ActAs.valueOption.map { actAs =>
+      val lifetime =
+        BigQuerySysProps.ImpersonationLifetimeSec.valueOption.flatMap(x => Try(x.toInt).toOption)
```

Comment:
do we want to silently default the `lifetime` if invalid ? can't we let it crash ?

Reply:
I think you are right